scopes/docs/docs/overview/overview.tsx

```diff
@@ -59,5 +70,32 @@ export function Overview({ titleBadges }: OverviewProps) {
     );
   }
 
-  return <ComponentPreview component={component} style={{ width: '100%', height: '100%' }} previewName="overview" />;
+  return (
+    <>
+      {laneId ? (
+        <div className={styles.overviewWrapper}>
```

Comment:
this might be legacy (we are switching to a separate overview outside the iframe)
consult with @GiladShoham, it might be irrelevant here

Reply:
@GiladShoham I see this being rendered on a workspace. Can you confirm if this is still needed?